server/src/main/java/org/opensearch/bootstrap/Bootstrap.java

```diff
@@ -195,6 +195,14 @@ private void setup(boolean addShutdownHook, Environment environment) throws Boot
             BootstrapSettings.CTRLHANDLER_SETTING.get(settings)
         );
 
+        SecureRandomInitializer.init();
+
+        var cryptoStandard = System.getenv("OPENSEARCH_CRYPTO_STANDARD");
+        if (cryptoStandard != null && cryptoStandard.equals("FIPS-140-3")) {
```

Comment:
nit: I've seen this pattern repeated a few places. I believe it could be simplified to `if ("FIPS-140-3".equals(cryptoStandard)) { ... }`.